doc/gridequivalent/gridequivalent_example.rst

```diff
@@ -0,0 +1,101 @@
+.. _gridequivalentexample:
+
+#############################
+Grid Equivalent Example
+#############################
+
+Here is ein example of what you can do:
+
+Create Example Network
+--------------------------------------------------
+
+First, we create a example network.
+
+.. code:: python
+
+	import pandapower as pp
+
+	net = pp.case9()
+    
+	# getting power flow results
+	pp.runpp(net)
+    
+    
+Define Grid Areas
+--------------------------------------------------
+The created network is shown below. Assuming that, it is divided into two subsystems. The buses (red) [0, 3, 4, 8] belong to the internal subsystem, which should be remained, and the external subsystem consists of buses (green) [1, 2, 5, 6, 7] that are going to be reduced. The boundary buses are [4, 8] belonging to the internal subsystem.
+
+.. image:: /pics/gridequivalent/full_case9.png
+	:width: 42em
+	:alt: alternate Text
+	:align: center
+    
+
+Call Grid Equivalent Calculation
+--------------------------------------------------
+
+According to the above assumptions, we call the Ward-equivalent calculation. The input variables are:
+
+.. code:: python
+
+	# equivalent type
+	eq_type = "ward"
+    
+	# internal buses: we don't need to give all internal buses to the function. Just one of them is enough.
+	internal_buses = [0]
+    
+	# boundary buses
+	boundary_buses = [4, 8]
+
+Then we start the Ward-equivalent calculation:
+
+.. code:: python
+
+	net_eq = pp.grid_equivalents.get_equivalent(net, eq_type, boundary_buses, internal_buses)
+
+
+Power Flow Results Comparison
+--------------------------------------------------
+We can compare the power flow results between the original grid "net" and the reduced grid "net_eq":
+
+.. code:: python
+
+	print("--- power flow (original grid) ---")
+	net.res_bus
+	print("--- power flow (reduced grid) ---")
+	net_eq.res_bus
+
+.. image:: /pics/gridequivalent/res_comparison.png
+	:width: 42em
+	:alt: alternate Text
+	:align: center
+
+It can be seen that the power flow results (**vm_pu**, **va_degree**) of the internal buses [0, 3, 4, 8] in both grids are the same, i.e., the equivalent calculation is successful. The **p_mw** and **q_mvar** values at the boundary buses [4, 8] are different due to the created Ward elements. The figure below shows the reduced grid.
```

Comment:
same -> the difference is below the tolerance

(if it is not actually the same, I assume there would be a numerical difference)